frontend/src/services/unsubscribe.service.ts

```diff
@@ -25,10 +23,15 @@ export async function unsubscribeRequest({
   }
 }
 
-function errorHandler(e: AxiosError, defaultMsg?: string): never {
+function errorHandler(e: unknown, defaultMsg?: string): never {
   console.error(e)
-  if (e.response && e.response.data && e.response.data.message) {
+  if (
+    axios.isAxiosError(e) &&
+    e.response &&
+    e.response.data &&
+    e.response.data.message
+  ) {
     throw new Error(e.response.data.message)
   }
-  throw new Error(defaultMsg || e.response?.statusText)
+  throw new Error(defaultMsg)
```

Comment:
Could you elaborate a bit more on this change? I'm a bit concerned about the fact that `defaultMsg` is optional and this is considered a breaking change

Reply:
I'll fix in a separate PR; found this one too.

Thanks for flagging!

<img width="760" alt="image" src="https://user-images.githubusercontent.com/67887489/179744537-d58929d6-0432-43b9-9c97-be68ee4342e3.png">